e2e-tests/page-objects/table-section.ts

```diff
@@ -0,0 +1,84 @@
+import { expect } from '@playwright/test';
+
+export class TableSection {
+  public readonly categoryHeading: Locator;
+  public readonly emissionsHeading: Locator;
+  public readonly noEstimationsText: Locator;
+  public readonly upstreamEmissionsEstimate: Locator;
+  public readonly employeeHardware: Locator;
+  public readonly serversStorageHardware: Locator;
+  public readonly networkingInfrastructurehardware: Locator;
+  public readonly directEmissionsEstimate: Locator;
+  public readonly employeeDevices: Locator;
+  public readonly serversStorage: Locator;
+  public readonly networkingInfrastructure: Locator;
+  public readonly indirectEmissionsEstimate: Locator;
+  public readonly cloudServices: Locator;
+  public readonly downstreamEmissionsEstimate: Locator;
+  public readonly endUserDevices: Locator;
+  public readonly networkDataTransfer: Locator;
+  public readonly totalEmissionsEstimate: Locator;
+  public readonly kilogramsColumn: Locator;
+  public readonly percentageColumn: Locator;
+
+  constructor(public page: Page) {
+    this.categoryHeading = page.getByRole('gridcell', { name: 'Category' });
+    this.emissionsHeading = page.getByRole('gridcell', { name: 'Emissions' });
+    this.noEstimationsText = page.getByRole('gridcell', { name: 'No estimation available' });
+    this.upstreamEmissionsEstimate = page.getByRole('gridcell', { name: 'Upstream Emissions Estimate', exact: true });
+    this.employeeHardware = page.getByRole('gridcell', { name: 'Employee Hardware', exact: true });
+    this.serversStorageHardware = page.getByRole('gridcell', { name: 'Servers and Storage Hardware', exact: true });
+    this.networkingInfrastructurehardware = page.getByRole('gridcell', {
+      name: 'Networking and Infrastructure Hardware',
+      exact: true,
+    });
+
+    this.directEmissionsEstimate = page.getByRole('gridcell', { name: 'Direct Emissions Estimate', exact: true });
+    this.employeeDevices = page.getByRole('gridcell', { name: 'Employee Devices', exact: true });
+    this.serversStorage = page.getByRole('gridcell', { name: 'Servers and Storage', exact: true });
+    this.networkingInfrastructure = page.getByRole('gridcell', { name: 'Networking and Infrastructure', exact: true });
+    this.indirectEmissionsEstimate = page.getByRole('gridcell', { name: 'Indirect Emissions Estimate', exact: true });
+    this.cloudServices = page.getByRole('gridcell', { name: 'Cloud Services', exact: true });
+    this.downstreamEmissionsEstimate = page.getByRole('gridcell', {
+      name: 'Downstream Emissions Estimate',
+      exact: true,
+    });
+    this.endUserDevices = page.getByRole('gridcell', { name: 'End-User Devices', exact: true });
+    this.networkDataTransfer = page.getByRole('gridcell', { name: 'Network Data Transfer', exact: true });
+    this.totalEmissionsEstimate = page.getByRole('gridcell', { name: 'Total Emissions Estimate', exact: true });
+    this.kilogramsColumn = page.locator('td:nth-child(2)');
```

Comment:
Given there are tables in the assumptions section, I think these should be more specific. Either check they are visible or that they are children of the carbon-estimation-table element (Possibly not ideal but I'd hope the element name should be pretty stable, there's not really anything else I can think of to identify a unique parent).

Reply:
Reading a little on Playwright's recommendations, the best way to identify it might be:

```suggestion
    this.kilogramsColumn = page.getByRole('treegrid').locator('td:nth-child(2)');
```

Which is something we added along with the keyboard support to open and close the sections.